pom.xml

```diff
@@ -42,7 +42,7 @@
     <maven.compiler.target>1.8</maven.compiler.target>
     <sourceReleaseAssemblyDescriptor>source-release-zip-tar</sourceReleaseAssemblyDescriptor>
 
-    <target.gen.source.path>${project.basedir}/target/generated-sources</target.gen.source.path>
+    <target.gen.source.path>${project.build.directory}/generated-sources</target.gen.source.path>
     <proto.cas.path>${project.basedir}/src/main/protobuf/</proto.cas.path>
```

Comment:
is this path correct ? I don't see any `src/main/protobuf` path ?

Also there are other places like line [549](https://github.com/apache/drill/pull/1746/files#diff-600376dffeb79835ede4a0b285078036R549) which can be modified to replace `target/classes` with `${project.build.outputDirectory}`